.travis.yml

```diff
@@ -115,7 +115,7 @@ matrix:
         - unzip -Pinfected al-khaser.bin.zip
         - cd $TRAVIS_BUILD_DIR/tests
       script:
-        - cmd.exe //C '.\test_pe.bat' 
```

Comment:
We need to change this  back ?

Reply:
Timeout increased for crush debugging. Revert to orig.

Reply:
See commit c5bd30a.